cedar-policy/src/api.rs

```diff
@@ -3617,3 +3618,36 @@ pub fn eval_expression(
         eval.interpret(&expr.0, &ast::SlotEnv::new())?,
     ))
 }
+
+/// Get a tuple of valid principal [`EntityTypeName`]s, action [`EntityUid`]s, and resource [`EntityTypeName`]s.
+pub fn get_valid_request_pars(
```

Comment:
I assume `pars` = principals, actions, resources? This might not be obvious to others, so maybe a name like `get_valid_request_types` would be better. Alternatively, we could provide separate functions: `get_valid_request_principal_types`, `get_valid_request_actions`, `get_valid_request_resource_types`

Reply:
Also: maybe these could be methods for the `Policy` type?

Reply:
The return values are not all entity names (actions are entity uids). So maybe `get_valid_request_envs`?

Reply:
> Also: maybe these could be methods for the `Policy` type?

I don't have any preference. @hakanson What do you think?

Reply:
I agree that this should be a method on `Policy`

Reply:
Renamed and refactored to a method.